addon/helpers/reorder.js

```diff
@@ -30,11 +33,49 @@ const OVERSHOOT = 2;
   @return {Promise}
 */
 
-export function reorder(app, mode, itemSelector, ...resultSelectors) {
+export async function reorder(mode, itemSelector, ...resultSelectors) {
+  await settled();
+
+  resultSelectors.forEach(async (selector, targetIndex) => {
```

Comment:
I believe we can't use `Array.prototype.forEach` here since it can't `await` async operations. https://github.com/babel/babel/issues/909